cmake/config.cmake

```diff
@@ -193,7 +193,6 @@ set(OMR_NOTIFY_POLICY_CONTROL OFF CACHE BOOL "TODO: Document")
 
 set(OMR_ENV_GCC OFF CACHE BOOL "TODO: Document")
 
-
-set(OMR_OPT_CUDA OFF CACHE BOOL "TODO: Document")
+set(OMR_OPT_CUDA ${CUDA_FOUND} CACHE BOOL "Enable CUDA support in OMR")
```

Comment:
If some forces CUDA off via a cache or command line option will this override it?

Reply:
Yep

Reply:
hmmm I think it is incorrect for an automatic detection to override a cache/command line option

Reply:
Oh sorry, I misinterpreted your sentence. The user-provided value would be used, overriding the automatically determined default value.

Reply:
ok sounds much better.